src/ms-review.tex

```diff
@@ -510,121 +489,149 @@ \section{Gamma-ray Data Analysis}
 background models can be subject to significant systematic uncertainties,
 these models can be improved by taking into account actual data
 from regions without known \gammaray sources. This includes methods 
-such as the ring or the FoV background techniques or
-background measurements performed within, e.g. reflected regions~\citep{Berge07}.
+such as the ring or the FoV background techniques or background measurements
+performed within, e.g. reflected regions~\citep{Berge07}.
+
 Data measured at the FoV or energy boundaries of the instrument are typically
 associated with a systematic uncertainty in the IRF. For this reason this part 
 of the data is often excluded from subsequent analysis by defining regions of
- \enquote{safe} data in the spatial as well as energy dimension.
-
-%% CN: for 1D analysis datasets, DL4 can actually be written as OGIP files,
-%% so as reduced datasets in a general format, not specific to Gammapy.
-%% Maybe you can briefly mention that this is due to the fact that the 1D Poisson
-%% spectral analysis is liong consilidated, but this also means that Gammapy 's
-%% dataset can handle lower energy data, e.g. X-ray.
-%% We also said in the abstract that Gammapy is compliant with other specifications
-%% adopted in high-energy astrophysics.
-The counts data and the reduced IRFs in the form of maps are bundled into datasets
-that represent the fourth data level (DL4). These reduced datasets can be written to disk,
-in a format specific to \gammapy to allow users to read them back at any time later
-for modelling and fitting. Different variations of such datasets support different 
-analysis methods and fit statistics. The datasets can be used to perform a joint-likelihood
-fit, allowing one to combine different measurements, e.g. from different observations
-but also from different instruments or event classes. They can also be used for binned
-simulation as well as event sampling to simulate DL3 events data.
-
-The next step is then typically to model and fit the datasets, either
-individually, or in a joint likelihood analysis. For this purpose \gammapy
-provides a uniform interface to multiple fitting backends. In addition to
-providing a variety of built-in models, including spectral,
-spatial and temporal model classes to describe the \gammaray emission in the sky,
-custom user-defined models are also supported.
-Spectral models can be simple analytical models or more complex ones from radiation
-mechanisms of accelerated particle populations (e.g. inverse Compton or $\pi^{0}$ decay).
-Independently or subsequently to the global modelling, the data can be
-re-grouped to compute flux points, light curves and flux maps as well as significance
-maps in different energy bands.
+\enquote{safe} data in the spatial as well as energy dimension.
+
+All the code related to reduce the DL3 data to binned data structures 
+is contained in the \code{gammapy.makers} sub-package.
+
+
+ \subsubsection{DL4: Binned Data Structures}
```

Comment:
Why not making this a subsection. It is strange to have one subsubsection only

Reply:
Indeed this happened by mistake. Fixed.